packages/cli-main/src/cli/commands/upgrade.ts

```diff
@@ -86,10 +85,10 @@ export default class Upgrade extends Command {
     const packagesToUpdate = packages.filter((pkg: string): boolean => {
       const pkgRequirement: string | undefined = deps[pkg]
       return Boolean(pkgRequirement)
-    })
+    }).map((pkg) => { return {name: pkg, version: 'latest'} })
```

Comment:
This needs to be updated because the method accepts an array of `DependencyVersion`s, not strings.